src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,26 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
-public class HelloWorld {
+import java.util.Random;
+
+public class HelloWorld extends CircleSupplier {
+    public static void main(String[] args) {
+        final Random random = new Random();
+        Circle circle = new Circle();
+        IsoscelesTrapezoid isoscelesTrapezoid = new IsoscelesTrapezoid();
+        Rectangle rectangle = new Rectangle();
+        RightTriangle rightTriangle = new RightTriangle();
+        Square square = new Square();
+        Figure[] choise = {circle, isoscelesTrapezoid, rectangle, rightTriangle, square};
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = choise[random.nextInt(choise.length)];
+            figures[i].getArea();
+            figures[i].draw();
```

Comment:
1. You're creating multiple objects even though you won't use most of them.
2. You're generating values inside figures instead of generating them outside and passing to constructor.

Use a proper statement that will create object only when needed. A hint: `switch` could be useful.